### DISCUSSION TITLE
Why The Style Manager Renders Multiple Times

### DISCUSSION THREAD

**Discussion by AgnRakul** [GrapesJS/grapesjs#4825] [Q&A]
![image](https://user-images.githubusercontent.com/70213104/210711958-01bc6c59-ddeb-48c0-ae91-46fb5d35ffc2.png)

How to Fix This Issues

**Comment by THEWebDegenerate** [comment]
That’s cold lol

**Reply by AgnRakul** [reply]
Fixed Buddy

**Reply by glaprida** [reply]
How did you fixed it? It happens to me too when building with react and placing the different panels manually with appendTo

**Comment by artf** [comment]
I guess that might be related to your configuration and without a reproducible demo it's hard to understand the issue
